.github/workflows/code-scan.yml

```diff
@@ -0,0 +1,20 @@
+name: "Code scanning"
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+  schedule:
+    - cron: "26 14 * * 2"
```

Comment:
why does the cron-schedule look so weird? Does this job run every tuesday at 14:26? why?